config-env.ts

```diff
@@ -0,0 +1,41 @@
+import { writeFile } from 'fs';
+
+const colors = require('colors');
+require('dotenv').config();
+
+const environment = process.env.ENVIRONMENT;
+let apiURL;
+let targetPath;
+
+if (environment === 'staging') {
+  apiURL = process.env.BASEAPIURL_STAGING;
+  targetPath = `./src/environments/environment.staging.ts`;
+} else if (environment === 'dev') {
+ apiURL = process.env.BASEAPIURL_DEV;
+ targetPath = `./src/environments/environment.ts`;
+}
+
+const envConfigFile = `
```

Comment:
Est ce que c'est une erreur ca? Tu na pas fermer
```suggestion
const envConfigFile = ``
```

Reply:
non ce n'est pas une erreur. Il s'agit d'un long string

Reply:
on dirait que tu la pas terminer.. https://stackoverflow.com/questions/35225399/typescript-angular-multiline-string

Reply:
sisi il est termine regarde  la ligne 30

Reply:
AHHHHHHHHHH je vois!